### DISCUSSION TITLE
Rest API deployment output files

### DISCUSSION THREAD

**Discussion by HugoB2409** [vercel/community#1455] [Help] [answered]
### Summary

I would like to access a deployment output files(files generated during build) via the rest api. I can get the source file using this endpoint /v6/deployments/{id}/files but i didn't find documentation on output files. On vercel.com you use https://vercel.com/api/file-tree/{deployment_url}?base=out. Is it a endpoint only for vercel.com or can i use it? Can i get the list of files recursively like with the /v6/deployments/{id}/files endpoint ?

Thanks

**Comment by amyegan** [comment, marked as answer]
Hi, Hugo! As you mentioned, the [Vercel API](https://vercel.com/docs/rest-api#) endpoint to [list Deployment files](https://vercel.com/docs/rest-api#endpoints/deployments/list-deployment-files) is `api.vercel.com/v6/deployments/{id}/files`, but an output files endpoint is not provided. Because `vercel.com/api/file-tree/{deployment_url}?base=out` is not an official part of the API, you could think of it as a use-at-your-own-risk endpoint. It may change or disappear at any time without warning
